harness/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include determined/deploy/aws/templates/*.yaml
+include determined/deploy/local/*.yaml
+include determined/deploy/local/configuration/*.yaml
```

Comment:
can we remove this line?  There hasn't been any such files for a while.

Reply:
done.